pkg/sloop/webserver/webserver.go

```diff
@@ -99,8 +100,8 @@ func webFileHandler(currentContext string) http.HandlerFunc {
 			logWebError(nil, "Not allowed", r, w)
 			return
 		}
-		fullPath := path.Join(prefix, fixedUrl)
-		data, err := readWebfile(fullPath, &afero.Afero{afero.NewOsFs()})
+		fullPath := common.GetFilePath(prefix, fixedUrl)
```

Comment:
if `fullPath` still needed?

Reply:
yes still needed, i moved it right above the line it's being used.